com.unity.render-pipelines.high-definition/Documentation~/Custom-Post-Process.md

```diff
@@ -151,6 +151,11 @@ Shader "Hidden/Shader/GrayScale"
         UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
 
         uint2 positionSS = input.texcoord * _ScreenSize.xy;
+
+        //NOTE: As of 12.x beta, for new custom post processes to be compatible with Dynamic resolution, please use:
```

Comment:
I think we need to put that in the custom post process template too (file: `CustomPostProcessingShader.template`)

Reply:
Also, please hyphenate post-processes :D:

//NOTE: As of 12.x beta, for new custom post-processes to be compatible with Dynamic resolution, please use:

Reply:
> I think we need to put that in the custom post process template too (file: `CustomPostProcessingShader.template`)

Updated the template file. Good catch!